classroom.rb

```diff
@@ -0,0 +1,18 @@
+# classroom.rb
+
+# Class representing a Classroom
+class Classroom
+  attr_accessor :label, :students
+
```

Comment:
- [ ] Let's ensure that the student class's property in the classroom remains read-only. We can accomplish this by using attr_reader :students instead of attr_accessor. This way, we maintain the intended behavior! 😊👍